blocked-edges/4.3.1.yaml

```diff
@@ -0,0 +1,4 @@
+to: 4.3.1
+from: 4\.2\..*
+# 4.2 -> 4.3 updates occasionally hit RequiredPoolsFailed, fixed in 4.2.18 and rc.0, but not in 4.2.16: https://bugzilla.redhat.com/show_bug.cgi?id=1782152 https://bugzilla.redhat.com/show_bug.cgi?id=1782149
+# 4.2 -> 4.3 updates on GCP occasionally have unreachable workloads: https://bugzilla.redhat.com/show_bug.cgi?id=1793635
```

Comment:
This is still NEW.  If it was grounds for blocking an edge, it would be grounds for blocking all 4.2.* -> 4.3.* edges.  Do we actually have any issues with 4.2.18 -> 4.3.1?  4.3.2 and 4.3.3 block _all_ incoming edges, but I haven't run down the three bugs they cite to see when they were introduced and when they were fixed.

Reply:
/shrug
There was a cluster of bugs fixed in 4.3.5 that led us to believe it was the first reliable upgrade path from 4.2.*. I've lost track of specifics and it doesn't warrant digging that deep now in my opinion. I'll update this.

Reply:
> I've lost track of specifics and it doesn't warrant digging that deep now in my opinion.

This is why we should be documenting the specifics.  I'll see about filing a parallel PR expanding on [these links][1].

[1]: https://github.com/openshift/cincinnati-graph-data/blob/4f4145dc35d44e0eccc4ec3c27d4d0aee0d69e1a/blocked-edges/4.3.3.yaml#L1